src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/ObjectSynchronizer.java

```diff
@@ -93,47 +79,34 @@ public static Iterator objectMonitorIterator() {
 
   private static class ObjectMonitorIterator implements Iterator {
 
-    // JVMTI raw monitors are not pointed by gBlockList
+    // JVMTI raw monitors are not pointed by _in_use_list
     // and are not included by this Iterator. May add them later.
```

Comment:
Perhaps:
// JVMTI raw monitors are not included in _in_use_list and
// are not returned by this Iterator.

Raw Monitors don't have the same semantics as ObjectMonitors
so I don't think returning them in the future is reasonable.